R/1-1-0check_input.R

```diff
@@ -2,7 +2,8 @@
 #'
 #' This function checks the input arguments for the \code{rdlearn} function.
 #'
-#' @param y A character string representing the name of the outcome variable.
+#' @inheritParams
+#' y A character string representing the name of the outcome variable.
```

Comment:
I think you need a function name to inherit parameter arguments, for example, `#' @inheritParams funcA`. If you need to keep the description for `y`, probably better to keep `#' @param`.